about.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="css" href="stylesheets/styles.css">
+    <!-- for FF, Chrome, Opera -->
+    <link rel="icon" type="image/png" href="./images/chi.png" sizes="16x16">
+    <link rel="icon" type="image/png" href="./images/chi.png" sizes="32x32">
+    <!-- for IE -->
+    <link rel="icon" type="image/x-icon" href="./images/chi.png" >
+    <link rel="shortcut icon" type="image/x-icon" href="./images/chi.png"/>
+  </head>
+  <body>
+    <header>
```

Comment:
currently all of your headers are empty. I would nest your nav within the header element.